app/design/frontend/Gorban/Sergey/web/css/source/_variables.less

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
There is more content in the original file from the Magento/luma theme than you have here: https://github.com/magento/magento2/blob/2.2.6/app/design/frontend/Magento/luma/web/css/source/_variables.less
Need to preserve the original content unless you really want to remove something

Reply:
My parent theme is blank, so I used this file https://github.com/magento/magento2/blob/2.2.6/app/design/frontend/Magento/blank/web/css/source/_variables.less.
I removed this file because I use font variables in _extend.less file now.